data/json/monsters/mammal.json

```diff
@@ -2351,5 +2351,62 @@
     "death_function": [ "NORMAL" ],
     "zombify_into": "mon_zolf",
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "KEENNOSE" ]
+  },
+  {
+    "id": "mon_reindeer",
+    "type": "MONSTER",
+    "name": { "str": "Reindeer" },
+    "description": "Also known as the Caribou, this deer is an icon of Christmas tradition.",
```

Comment:
It's a bit weird to reference Christmas here. This is not a mmorpg with seasonal events.